### DISCUSSION TITLE
Must install vitest in global?

### DISCUSSION THREAD

**Discussion by ShortArrow** [marilari88/neotest-vitest#18] [Q&A] [answered]
Is Specification of neotest-vitest has specified as must install vitest in global?

neotest-vitest was not working, when add vitest to only dev-dependencies of project.
This is [sample](https://github.com/ShortArrow/vitest-react-uitest-sample).

**Comment by marilari88** [comment, marked as answer]
hi @ShortArrow, you don't need to install vitest globally. This adapter enables test discovery also with vitest installed as dev dependency. In fact I tried the repo you linked and it works fine. 
Can you tell more about your issue? Are you able to discovery tests? do you see any error (type `:messages`)? What's your plugin configuration?

**Reply by ShortArrow** [reply]
Thank you! I am happy by know that fact don't need to install vitest globally. 
After this, I was detect confliction about neotest-vim-test.

So, my fix is this. 

```lua
require("neotest").setup{
  adapters = {
    require("neotest-vitest"),
    require("neotest-vim-test")({
      ignore_file_types = { "vitest" },
    }),
  }
}
```

And run `yarn global remove vitest`.
`neotest-vitest` had behaviors that I was expected.
